### PR TITLE
multi: mailbox metrics

### PR DESCRIPTION
**Pull request by ellemouton**
Extract prometheus config so that it can be used more generally & get metrics for number of active mailboxes

### CODE REVIEW

**Comment by ellemouton** [comment]
cc @guggero

**Review by guggero — COMMENTED** [review]
tACK, very nice! Just a nit and a suggestion for even more stats.

File: sample-conf.yaml [hunk 1]
```diff
@@ -151,3 +151,9 @@ hashmail:
   enabled: true
   messagerate: 20ms
   messageburstallowance: 1000
+
+# Enable the prometheus metrics exporter so that a prometheus server can scrape
+# the metrics.
+prometheus:
```
Comment (ellemouton) [inline sample-conf.yaml:157]: also have now added this

File: prometheus.go [hunk 2]
```diff
@@ -29,6 +38,7 @@ func StartPrometheusExporter(cfg *PrometheusConfig) error {
 	}
 
 	// Next, we'll register all our metrics.
+	prometheus.MustRegister(mailboxCount)
```
[the remark this answers is not in the document]
Comment (ellemouton) [reply prometheus.go:41]: cool, added in next commit

**Review by ellemouton — COMMENTED** [review]
Thanks for the review @guggero 🚀 
I have updated and also added the prometheus stuff to the sample-conf file

**Review by guggero — APPROVED** [review]
tACK, LGTM :tada:

**Pull request merged**